recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ source:
   path: .
 
 build:
-  number: 0
+  number: 1
```

Comment:
I prefer to always bump the `CalVer` above and just leave the build number `0` forever. It would avoid confusions like this and will always help us spot the version/release date immediately.